docs/generation-WIP/configuration/sample_port_configs.md

```diff
@@ -0,0 +1,98 @@
+# HPE Port configurations and standards - CSM v1.2
+
+Hardware Type: HPE
+
+Software Version: CSM v1.2
+
+# NCN-Master port config
+```
+```
+
+# NCN-Worker port config
+```
+
+interface lag  multi-chassis
+    no shutdown
+    description 
+    no routing
+    vlan trunk native 1
+    vlan trunk allowed 1-2,4,7
+    lacp mode active
+    lacp fallback
+    spanning-tree port-type admin-edge
+
+interface 1/1/2
+    no shutdown
+    mtu 9198
+    description 
+    lag ```
+
+# NCN-Storage port config
+```
+
+interface lag  multi-chassis
+    no shutdown
+    description 
+    no routing
+    vlan trunk native 1    vlan trunk allowed 10    lacp mode active
```

Comment:
new line?

Reply:
interestingly enough i just moved the files and repointed to templates etc... templates would have changed.

Reply:
fixed.